determined-ai/determined
# chore: split `resource pool` from Store Context

## Description

[WEB-587]



## Test Plan
- Check if `resource pool` is loaded in 
  - Cluster page
  - Job Queue page
  - Launch jupyter modal page
  - Hyper Parameter modal



## Commentary (optional)




## Checklist
See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.





[WEB-587]: https://determinedai.atlassian.net/browse/WEB-587?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
